single_subject/batch_single_subject.sh

```diff
@@ -38,11 +38,18 @@ fi
 
 # Go to T2 contrast
 cd data/t2
-# Spinal cord segmentation (
-sct_deepseg_sc -i t2.nii.gz -c t2 -qc ~/qc_singleSubj
+# Spinal cord segmentation (using new 2024 method)
+sct_deepseg -task seg_sc_contrast_agnostic -i t2.nii.gz -qc ~/qc_singleSubj
+# You can also choose your own output filename using the “-o” argument
```

Comment:
maybe we can mention here that the default output has the suffix `_seg.nii.gz`? something like this?

```suggestion
# The default output is t2_seg.nii.gz
# You can also choose your own output filename using the “-o” argument
```